autograder/rest_api/serialize_ultimate_submission_results.py

```diff
@@ -86,6 +94,21 @@ def _get_submission_data_with_results(submission_fdbk: SubmissionResultFeedback,
     else:
         submission_results = submission_fdbk.to_dict()
 
+    if include_handgrading:
+        handgrading_result_available = (
+            hasattr(submission_fdbk.submission, 'handgrading_result') and (
+                submission_fdbk.submission.handgrading_result.finished_grading)
+        )
+
+        if handgrading_result_available:
+            handgrading_result = submission_fdbk.submission.handgrading_result
+            submission_results['handgrading_total_points'] = handgrading_result.total_points
+            submission_results['handgrading_total_points_possible'] = (
+                handgrading_result.total_points_possible)
+        else:
+            submission_results['handgrading_total_points'] = ''
```

Comment:
Why empty string instead of None?

Reply:
The goal was to have the CSV file have an empty entry on the Handgrading Total Points column for this case. If setting it equal to None does the same effect then I can do that instead.

Reply:
Makes sense. If None accomplishes the same thing, I think None has a slightly clearer meaning in the code and should be preferred.